README.md

```diff
@@ -75,18 +75,19 @@ The `name` field provides minimal frame attribution so that the observing frame
 The above covers existing use cases found in the wild, enables document-level attribution, and eliminates the negative performance implications mentioned earlier. To receive these notifications, the application can subscribe to them via PerformanceObserver interface:
 
 ```javascript
-var observer = new PerformanceObserver(function(list) {
-  var perfEntries = list.getEntries();
-  for (var i = 0; i < perfEntries.length; i++) {
+const observer = new PerformanceObserver(function(list) {
+  const perfEntries = list.getEntries();
```

Comment:
If you're modernizing the code, maybe switch to using `for...of` like referenced in this example? https://web.dev/custom-metrics/#long-tasks-api

Reply:
Done